src/Fonts/FontRegistry.php

```diff
@@ -32,9 +32,19 @@ public function findMatchingFont(?string $family, bool $italic, float $weight):
             }
         }
 
-        // Attempt to find the closest-weight match with correct family and italicness.
-        $match = $this->closestMatchBasedOnWeight(function (FontFile $font) use ($family, $anyFontFamily, $italic) {
-            return ($anyFontFamily || $font->getFamily() === $family) && $font->isItalic() === $italic;
+        // Attempt to find the closest-weight match with correct family and cursiveness.
+        $match = $this->closestMatchBasedOnWeight(function (FontFile $font) use ($family, $anyFontFamily, $style) {
+            $result = ($anyFontFamily || $font->getFamily() === $family); 
```

Comment:
There is a trailing space:

```suggestion
            $result = ($anyFontFamily || $font->getFamily() === $family);
```